src/styles/blocks/card.scss

```
@@ -0,0 +1,118 @@
+@use "../variables" as *;
+
+.card {
+    width: 200px;
+    padding: 32px 16px 16px;
+    display: flex;
+    flex-direction: column;
+    border: 1px solid $grey-color;
+    border-radius: 5px;
+
+    &:hover {
+        transform: scale(1.2);
+        transition: 0.3s;
+    }
+
+    &__image {
+        width: 160px;
+        height: 134px;
+        background-repeat: no-repeat;
+        background-size: contain;
+        align-self: center;
+    }
+
+    &__nameProduct {
+        font-size: 12px;
+        font-weight: 500;
+        margin-top: 40px;
+        line-height: 18px;
+        color: $main-color;
+        letter-spacing: 0;
+    }
+
+    &__codeProduct {
+        margin: 4px 0;
+        font-size: 10px;
+        font-weight: 400;
+        line-height: 14px;
+        color: $secondary-color;
+    }
+
+    &__stars {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        margin-top: 12px;
+    }
+
+    &__reviews {
+        font-size: 10px;
+        font-weight: 400;
+        text-align: center;
+        color: $main-color;
+    }
+
+    &__price {
+        display: flex;
+        justify-content: space-between;
+        margin: 24px 0 16px;
+
+        &-name {
+            font-size: 12px;
+            font-weight: 400;
+            line-height: 18px;
+            color: $secondary-color;
+            text-align: left;
+        }
+
+        &-value {
+            font-size: 16px;
+            font-weight: 700;
+            line-height: 18px;
+            text-align: right;
+            color: $main-color;
+        }
+    }
+
+    &__button {
+        height: 40px;
+        background-color: $blue-color;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        border-radius: 5px;
+        border: 1px solid transparent;
+        text-decoration: none;
+        font-weight: 700;
+        font-size: 14px;
+        line-height: 16px;
+        color: $white-color;
+
+
+        &:hover {
+            background-color: $white-color;
+            border-radius: 5px;
+            color: $blue-color;
+            border: 1px solid $blue-color;
+            transition: 0.3s;
+        }
+    }
+}
+
+@media screen and (min-width: 488px) {
+    .container {
+        grid-template-columns: repeat(2, $card-width);
+    }
+}
+
+@media screen and (min-width: 768px) {
+    .container {
+        grid-template-columns: repeat(3, $card-width);
+    }
+}
+
+@media screen and (min-width: 1024px) {
+    .container {
+        grid-template-columns: repeat(4, $card-width);
+    }
+}
```

it should be in container.scss and try to use nesting media quesries
```suggestion
.container {
  @media screen and (min-width: 488px) {...}
  @media screen and (min-width: 768px) {...}
  @media screen and (min-width: 1024px) {...}
}
```

---

moved to container and nesting done